lib/vanagon/platform/windows.rb

```diff
@@ -289,6 +289,105 @@ def output_dir(target_repo = "")
         File.join("windows", target_repo, @architecture)
       end
 
+      # Generate correctly formatted wix elements that match the
+      # structure of the directory input
+      #
+      # @param services, Array of components services
+      # and optionally:
+      # @return [string] correctly formatted wix element string
+      def generate_wix_dirs(services)
+        directories = []
+        services.map { |svc| directories.push({ :path => svc.service_file, :id => "#{svc.id}BINDIR" }) }
```

Comment:
`service_file` corresponds to `"SourceDir\\#{settings[:base_dir]}\\#{settings[:company_id]}\\#{settings[:product_id]}\\puppet\\bin\\nssm.exe"`, right?